scalalib/test/resources/classloader-test/test/src/ClassLoaderTests.scala

```diff
@@ -0,0 +1,13 @@
+package mill.scalalib
+
+import utest._
+
+object ClassLoaderTests extends TestSuite {
+  try {
+    getClass().getClassLoader().loadClass("com.sun.nio.zipfs.ZipFileSystemProvider")
+  } catch {
+    case _: ClassNotFoundException if !System.getProperty("java.specification.version").startsWith("1.") =>
+      // Don't fail on Java 9+
+  }
+  val tests = Tests {}
```

Comment:
a bit nitpicking, but wouldn't it be nicer to read, when we move the part that can fail into a test?

Reply:
Refactored the test. 👍

Reply:
Nice